### PR TITLE
llcppg/ast:field access & static

### PR DESCRIPTION
**Pull request by luoliwoshang**

### CODE REVIEW

File: chore/llcppg/ast/ast.go [hunk 1]
```diff
@@ -218,10 +218,12 @@ func (*CommentGroup) exprNode() {}
 // ------------------------------------------------
 
 type Field struct {
-	Doc     *CommentGroup // associated documentation; or nil
-	Type    Expr          // field/method/parameter type; or nil
-	Names   []*Ident      // field/method/(type) parameter names; or nil
-	Comment *CommentGroup // line comments; or nil
+	Doc      *CommentGroup   // associated documentation; or nil
+	Access   AccessSpecifier // field access(Record Type); Struct Field default is Public,Class Field default is Private
```
Comment (xushiwei) [inline chore/llcppg/ast/ast.go]: add these properties to the end.

Comment (luoliwoshang) [reply chore/llcppg/ast/ast.go]: updated

File: chore/llcppg/ast/ast.go [hunk 2]
```diff
@@ -218,10 +218,12 @@ func (*CommentGroup) exprNode() {}
 // ------------------------------------------------
 
 type Field struct {
-	Doc     *CommentGroup // associated documentation; or nil
-	Type    Expr          // field/method/parameter type; or nil
-	Names   []*Ident      // field/method/(type) parameter names; or nil
-	Comment *CommentGroup // line comments; or nil
+	Type     Expr            // field/method/parameter type; or nil
+	Names    []*Ident        // field/method/(type) parameter names; or nil
+	IsStatic bool            // static field
```
Comment (xushiwei) [inline chore/llcppg/ast/ast.go]: add this to the end

File: chore/llcppg/ast/ast.go [hunk 3]
```diff
@@ -218,10 +218,12 @@ func (*CommentGroup) exprNode() {}
...
+	Comment  *CommentGroup   // line comments; or nil
+	Doc      *CommentGroup   // associated documentation; or nil
```
Comment (xushiwei) [inline chore/llcppg/ast/ast.go]: don't change this.

Comment (luoliwoshang) [reply chore/llcppg/ast/ast.go]: Updated, I misunderstood the new line here

**Pull request merged**